### PR TITLE
Kenza/backend requests

### PR DESCRIPTION
**Pull request by bkenza**
- Fetch ads
- Fetch ad by id
- Delete an ad
- Fetch general meetings
- Fetch admin meetings
- Fetch general meeting by id
- Fetch admin meeting by id

### CODE REVIEW

File: src/Components/AdminDashboard/AdminUsers.js [hunk 1]
```diff
@@ -20,9 +20,7 @@ function AdminUsers () {
    * Function that handles user registration
    * @param {*} e 
    */
-  const handleRegister = async (e) => {
-    e.preventDefault();
-
```
Comment (KonstH) [inline src/Components/AdminDashboard/AdminUsers.js:25]: Put back my `e.preventDefault()`, NOW

Comment (bkenza) [reply src/Components/AdminDashboard/AdminUsers.js:25]: OKAY IM SORRY

File: src/Components/CondoAssociation/Ads/components/PostAdButton.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Context as CondoAssociationContext } from '../../../../context/CondoAssociationContext';
```
Comment (KonstH) [inline src/Components/CondoAssociation/Ads/components/PostAdButton.js]: Pls remove this, as it is not used. Thank you very much for your attention

File: src/Components/CondoAssociation/Meetings/pages/Meetings.js [hunk 3]
```diff
@@ -59,21 +40,68 @@ function CondoMeetings () {
...
+  //           admin={admin}
+  //           title={title}
+  //           date={date}
+  //           // duration={duration}
+  //           description={description}
+  //         />
+  //       </Fragment>
+  //     );
+  //   });
+  // };
```
Comment (KonstH) [inline src/Components/CondoAssociation/Meetings/pages/Meetings.js]: pls clean up your nasty comments 🤮

File: src/Components/CondoAssociation/Meetings/pages/Meetings.js [hunk 4]
```diff
@@ -59,21 +40,68 @@ function CondoMeetings () {
...
+  const RenderGeneralMeetings = () => {
+    if (generalMeetings) {
+      return generalMeetings.map(({ id, title, created_at, description, agenda, resolution }) => {
+        return (
+          <Fragment key={id}>
+            <MeetingCard
+              id={id}
+              title={title}
+              created_at={created_at}
+              // duration={duration}
```
Comment (KonstH) [inline src/Components/CondoAssociation/Meetings/pages/Meetings.js]: 🤮

File: src/context/CondoAssociationContext.js [hunk 5]
```diff
@@ -95,17 +95,28 @@ const fetchDiscussion = dispatch => async ({ discussionId }) => {
...
+      url: 'http://localhost:8080/con/api/discussions/addDiscussion.php',
+      headers: {
+        'content-type': 'application/json'
+      },
+      data: { user_id, id, title, content, is_public } // add condo_assoc title
+    });
+    console.log(response);
+    return response;
+  }
+  // return data.id;
```
Comment (KonstH) [inline src/context/CondoAssociationContext.js]: remove the logs + comment

File: src/context/CondoAssociationContext.js [hunk 6]
```diff
@@ -213,52 +230,64 @@ const udpdateAd = dispatch => async ({ adId, condoAssociationId, title, contactN
...
-    await axios.delete('', { adId }); // DELETE ad URL
+    const response = await axios({
+      method: 'delete',
+      url: 'http://localhost:8080/con/api/ads/deleteAd.php',
+      headers: {
+        'content-type': 'application/json'
+      },
+      data: { condo_assoc_post_id, user_id: localStorage.getItem('userid') } // add condo_assoc title
+    });
+    console.log(response);
```
Comment (KonstH) [inline src/context/CondoAssociationContext.js]: logggg

**Review by KonstH — CHANGES_REQUESTED** [review]

File: src/Components/CondoAssociation/Meetings/pages/Meetings.js (hunk 3 above)
Comment (bkenza) [reply src/Components/CondoAssociation/Meetings/pages/Meetings.js]: - [ ] be nice

**Review by KonstH — APPROVED** [review]
🔥 🚒 🧨

**Pull request merged**